src/psyclone/tests/psyir/transformations/profile_test.py

```diff
@@ -276,42 +277,29 @@ def test_unique_region_names():
 
     # This regular expression puts the region names into groups.
     # Make sure even though the kernels have the same name, that
-    # the created regions have different names. In order to be
-    # flexible for future changes, we get the region names from
-    # the ProfileStart calls using a regular expressions (\w*
-    # being the group name enclosed in "") group. Python will store
-    # those two groups and they can be accessed using the resulting
-    # re object.group(n).
+    # the created regions have different names.
```

Comment:
Not your fault but "Make sure that the created regions have different names, even though the kernels have the same name."

Reply:
Better thanks.